src/layouts/home-assistant-main.js

```diff
@@ -126,6 +126,10 @@ class HomeAssistantMain extends NavigateMixin(EventsMixin(PolymerElement)) {
   _computeDisableSwipe(hass) {
     return NON_SWIPABLE_PANELS.indexOf(hass.panelUrl) !== -1;
   }
+
+  defaultPage() {
```

Comment:
Should be a property and it's internal so prefix name with underscore

Reply:
As there are only 2 methods in that class with an underscore, can the rest be considered not internal then?

Reply:
No, probably needs fixing (but in a different PR)

Reply:
Question is if the underscore here makes sense at all as none of the methods, including connectedCallback etc, is ever publicly accessed.

Reply:
connectedCallback is a public API as defined by custom elements. It is called by the browser.